docker/bin/create-release

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
What is this empty file for ?

Reply:
This was a placeholder for a development side piece that I accidentally placed in the wrong branch. Simply put, shouldn't have been here and has been removed.